docarray/array/array_stacked.py

```diff
@@ -101,6 +101,10 @@ def _create_columns(
                 ):
                     columns_fields.append(field_name)
 
+        if not columns_fields:
+            # nothing to stack
+            return {}
```

Comment:
but in your case why will the columns field be empty here ? Because it sill looks at the at the schema of the da (i.e, document_type) which should still contains tensor field